File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        # count=[0]*26
        # for x in s:
        #     count[ord(x)-97]+=1
        # for y in t:
        #     count[ord(y)-97]-=1
        # for value in count:
        #     if value!=0:
        #         return False
        # return True   
        sorted_s=sorted(s)
        sorted_t=sorted(t)
        return sorted_s==sorted_t             

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int []a=new int[26];
        if(s.length() != t.length()){
            return false;
        }
        for(int i=0;i<s.length();i++){
            a[s.charAt(i)-97]++;
            a[t.charAt(i)-97]--;
        }
        // for(int i=0;i<t.length();i++){
        //     a[t.charAt(i)-97]--;
        // }
        for(int i=0;i<26;i++){
            if(a[i]!=0) return false;
        }
        return true;
    }
}